packages/plugin-ext/src/hosted/node/scanners/scanner-theia.ts

```diff
@@ -165,6 +165,8 @@ export class TheiaPluginScanner implements PluginScanner {
                 for (const c of configurations) {
                     const config = this.readConfiguration(c, rawPlugin.packagePath);
                     if (config) {
+                        const keys = Object.keys(config.properties);
+                        keys.forEach(key => { config.properties[key].title = config.title; });
```

Comment:
Suggestion: We don't need to make it so complicated by first getting all the keys.
```suggestion
                        Object.values(config.properties).forEach(property => property.title = config.title);
```

Reply:
Good to me. I'll integrate this suggestion.